src/renderers/label.util.ts

```diff
@@ -67,10 +55,6 @@ const isRequired = (schema: JsonSchema, schemaPath: string): boolean => {
  */
 export const getElementLabelObject =
```

Comment:
This method is now obsolete, please remove it and export getLabelObject instead. You will need to update all the imports of course.